lib/src/components/NxDropdownMenu/NxDropdownMenu.tsx

```diff
@@ -4,28 +4,108 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React, { useLayoutEffect, forwardRef } from 'react';
+import React, { useLayoutEffect, forwardRef, useState, useEffect, useRef, KeyboardEventHandler } from 'react';
 import classnames from 'classnames';
+import useMergedRef from '@react-hook/merged-ref';
 
 import { Props, propTypes } from './types';
 
 import './NxDropdownMenu.scss';
 
 export { Props };
 
+const FOCUSABLE_MENU_ITEMS_SELECTOR = 'a:not([disabled]):not(.disabled), '
++ 'button:not([disabled]):not(.disabled), '
++ 'input:not([disabled]):not(.disabled)';
```

Comment:
I think this is the same
```suggestion
const FOCUSABLE_MENU_ITEMS_SELECTOR = ':is(a, button, input):not([disabled], .disabled])';
```

Reply:
Might not hurt to also prefix this with `:scope > `

Reply:
> Might not hurt to also prefix this with `:scope > `

Ooo I didn't know :is is supported with major browsers now, I think multiple argument :not is not supported with the current safari though, not 100% sure, going to double check.

Reply:
> Ooo I didn't know :is is supported with major browsers now, I think multiple argument :not is not supported with the current safari though, not 100% sure, going to double check.

MDN seems to say it's been supported since safari 9